ownCloudAppFramework/AppLock Settings/AppLockSettings.m

```diff
@@ -53,6 +54,7 @@ + (OCClassSettingsIdentifier)classSettingsIdentifier
 {
 	return (@{
 		OCClassSettingsKeyPasscodeEnforced : @(NO),
+		OCClassSettingsKeyRequiredPasscodeDigits : @(NO),
```

Comment:
I think this should probably be `OCClassSettingsKeyPasscodeEnforcedByDevice` here.